.github/workflows/performance.yml

```
@@ -0,0 +1,66 @@
+name: Performance Tests
+
+on:
+  push:
+    branches:
+      - trunk
+      - 'feature/**'
+
+jobs:
+  # Runs the performance test suite.
+  #
+  # Performs the following steps:
+  # - Checks out the repository.
+  # - Sets up Node.js.
+  # - Installs npm dependencies.
+  # - Logs Docker debug information (about the Docker installation within the runner).
+  # - Builds WordPress to run from the `build` directory.
+  # - Starts the WordPress Docker container.
+  # - Logs the running Docker containers.
+  # - Install WordPress within the Docker container.
+  # - Run the Performance tests.
+  # - Ensures version-controlled files are not modified or deleted.
+  performance:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == '10up/wordpress-develop' || github.event_name == 'push' }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Docker debug information
+        run: |
+          docker -v
+          docker-compose -v
+
+      - name: Build WordPress
+        run: npm run build
+
+      - name: Start Docker environment
+        run: npm run env:start
+
+      - name: WordPress Docker container debug information
+        run: |
+          docker-compose run --rm mysql mysql --version
+          docker-compose run --rm php php --version
+          docker-compose run --rm php php -m
+          docker-compose run --rm php php -i
+          docker-compose run --rm php locale -a
+
+      - name: Install WordPress
+        run: npm run env:install
+
+      - name: Run Performance tests
```

```suggestion
      - name: Run Performance tests
```

@joemcgill This should be removed because it causes problems in the workflow. [Check this](https://github.com/mukeshpanchal27/wordpress-develop/actions/runs/4062622621/jobs/6993899120)

---

@mukeshpanchal27 Hmm, not sure if that means we should remove the code. We probably _do_ need to install WordPress. Potentially something else is wrong in the Docker configuration that causes this error?

---

This step will need to run, and seems to be working fine in [the E2E checks](https://github.com/mukeshpanchal27/wordpress-develop/actions/runs/4063792131/jobs/6996440196) that this is based from. The only difference that I can spot between to two is that the performance check does not include the step that configured environment variables. I wonder if that is affecting permissions? 

See: https://github.com/mukeshpanchal27/wordpress-develop/actions/runs/4063792131/workflow#L55-L58

---

Thanks @joemcgill, That step would solve the issues. The tests passed after I updated the PR with the changes. 